doc/lua_api.txt

```diff
@@ -7948,6 +7950,11 @@ Used by `HTTPApiTable.fetch` and `HTTPApiTable.fetch_async`.
         multipart = boolean
         -- Optional, if true performs a multipart HTTP request.
         -- Default is false.
+        -- Post only use post_data
```

Comment:
and what is this supposed to mean?

Reply:
Multipart requests must use the post_data and can only be POST method